config/locales/en.yml

```diff
@@ -4,10 +4,16 @@ en:
     account_reset_cancellation_notice: Your request to delete your login.gov account has been cancelled.
     account_reset_notice: As requested, your login.gov account will be deleted in 24 hours. Don't want to delete your account? Sign in to your login.gov account to cancel.
     authentication_otp:
-      sms: 'Login.gov: Your security code is %{code}. It expires in %{expiration} minutes. Don''t share this code with anyone.'
+      sms: |-
+        Login.gov: Your security code is %{code}. It expires in %{expiration} minutes. Don't share this code with anyone.
+
+        @login.gov #%{code}
```

Comment:
do we need to change this for the sandbox envs that use `identitysandbox.gov`?

If so, I think we should interpolate `%{domain}` and pass it in like `domain: Identity::Hostdata.domain`

Reply:
> do we need to change this for the sandbox envs that use `identitysandbox.gov`?

Yeah, we should. Good call.

Reply:
Updated in 834b3207dd7767e46bd31ca027008425a9ede965.